GenderPayGap.WebUI/BackgroundJobs/ScheduledJobs/PurgeOrganisationsJob.cs

```diff
@@ -58,7 +58,8 @@ private List<Organisation> GetOrganisationsToPurge(DateTime deadline)
                 .Where(o => o.Created < deadline)
                 .Where(o => !o.Returns.Any())
                 .Where(o => !o.OrganisationScopes.Any(sc => sc.ScopeStatus == ScopeStatuses.InScope || sc.ScopeStatus == ScopeStatuses.OutOfScope))
-                .Where(o => !o.UserOrganisations.Any(uo => uo.Method == RegistrationMethods.Manual || uo.PINConfirmedDate != null || uo.PINSentDate > deadline))
+                .AsEnumerable()

```

Comment:
It might be worth adding a comment on this line explaining why we need to do `.AsEnumerable()`
Just so that someone doesn't remove it later by mistake (or knows if they are able to remove it)